src/components/notification/index.tsx

```diff
@@ -1,42 +1,21 @@
 import React, { CSSProperties } from 'react';
-import { TextParentStyles } from 'src/index';
-
-interface NotificationProps {
-  title: string;
-  description: string;
-  primaryButtonLabel?: string;
-  secondaryButtonLabel?: string;
-  primaryButtonStyle?: CSSProperties;
-  secondaryButtonStyle?: CSSProperties;
-  onClickPrimaryBtn?: () => void;
-  onClickSecondaryBtn?: () => void;
-  titleStyle?: CSSProperties;
-  textStyle?: CSSProperties;
-  primaryDisableBtnStyle?: CSSProperties;
-  secondaryDisableBtnStyle?: CSSProperties;
-  disablePrimaryBtn?: boolean;
-  disableSecondaryBtn?: boolean;
-  onClickView?: () => void;
-}
+import { EmbeddedManager, TextParentStyles } from 'src/index';
+import { EmbeddedMessageData } from '../types';
 
 /* Note: Add export to this const when support Embedded Message View Types in a later release. */
-const Notification: React.FC<NotificationProps> = ({
-  title,
-  description,
-  primaryButtonLabel,
-  secondaryButtonLabel,
-  primaryButtonStyle,
-  secondaryButtonStyle,
-  onClickPrimaryBtn,
-  onClickSecondaryBtn,
-  textStyle,
-  titleStyle,
-  primaryDisableBtnStyle,
-  secondaryDisableBtnStyle,
-  disablePrimaryBtn,
-  disableSecondaryBtn,
-  onClickView
-}) => {
+export const Notification = (props: EmbeddedMessageData) => {
```

Comment:
`IterableEmbeddedMessage` instead of `EmbeddedMessageData`?

Reply:
We can't. `IterableEmbeddedMessage` is responsible for determining the message type received, while `EmbeddedMessageData` contains the properties needed for custom styling of Out-of-the-Box (OOTB) views.